core/templates/filters/parameterize-rule-description.pipe.ts

```diff
@@ -65,13 +72,14 @@ export class ParameterizeRuleDescriptionPipe implements PipeTransform {
     var PATTERN = /\{\{\s*(\w+)\s*(\|\s*\w+\s*)?\}\}/;
     var iter = 0;
     while (true) {
-      if (!description.match(PATTERN) || iter === 100) {
+      let match = description.match(PATTERN);
```

Comment:
prefer `let/const` over var. Here and elsewhere

Reply:
Done